.github/workflows/scripts/crates.io-script.bash

```diff
@@ -0,0 +1,5 @@
+function get_crate_version() {
+    curl -L https://crates.io/api/v1/crates/$1/versions | jq -r '.versions[] | select( .yanked == false) | .num' | sort --version-sort | tail -n 1
+}
+
+export -f get_crate_version
```

Comment:
This script is for the bootstrap of the build process and is designed to be used to determine the installation of various other tools.

Since the scripts in `cargo-make` and` Makefile.toml` can be used after these determineed are made and the installation process is performed, it is necessary to make this script independent.

Reply:
Understood. Putting this script here should be better.

Other comments:

- Function name should better to be `get_crate_latest_version`
- Add comment to describe what the function does